### PR TITLE
Add README to Security Observability with eBPF folder

### PR DESCRIPTION
**Pull request by xmulligan**
Signed-off-by: Bill Mulligan <billmulligan516@gmail.com>

### CODE REVIEW

**Review by kevsecurity — APPROVED** [review]

**Review by sharlns — APPROVED** [review]
LGTM. 🚀

File: docs/security-observability-with-ebpf/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Security Observability with eBPF Examples
+
+This repo contains examples from the [Security Observability with eBPF](https://www.oreilly.com/library/view/security-observability-with/9781492096719/) book. They can be used by those wanting to follow along in the book or as examples for users wanting to create their own TracingPolicies.
```
Comment (xmulligan) [inline docs/security-observability-with-ebpf/README.md:3]:
```suggestion
This repo contains examples from the [Security Observability with eBPF](https://www.oreilly.com/library/view/security-observability-with/9781492096719/) book.
They can be used by those wanting to follow along in the book or as examples for users wanting to create their own TracingPolicies.
These examples were contributed by the report's authors @sharlns and @jedsalazar.
```

**Comment by xmulligan** [comment]
I just combined my wording and #465 since we both had the same idea

**Comment by jrfastab** [comment]
hmm @xmulligan I merged @lizrice updates which then collided with yours. Can you folks sort it out and rebase.

**Comment by xmulligan** [comment]
I'm also happy with Liz's PR so I'll just close this one

**Pull request closed without merging**